solr/solr-ref-guide/modules/deployment-guide/pages/circuit-breakers.adoc

```diff
@@ -48,19 +63,25 @@ To enable and configure the JVM heap usage based circuit breaker, add the follow
 
 The `threshold` is defined as a percentage of the max heap allocated to the JVM.
 
-It does not logically make sense to have a threshold below 50% and above 95% of the max heap allocated to the JVM.
+For the circuit breaker configuration, a value of "0" maps to 0% usage and a value of "100" maps to 100% usage.
+
+It does not logically make sense to have a threshold below 50% or above 95% of the max heap allocated to the JVM.
 Hence, the range of valid values for this parameter is [50, 95], both inclusive.
 
 Consider the following example:
 
 JVM has been allocated a maximum heap of 5GB (-Xmx) and `threshold` is set to `75`.
 In this scenario, the heap usage at which the circuit breaker will trip is 3.75GB.
 
-=== CPU Utilization
+=== System CPU Usage Circuit Breaker
+This circuit breaker tracks system CPU usage and triggers if the recent CPU usage exceeds a configurable threshold.
+
+This is tracked with the JMX metric `OperatingSystemMXBean.getSystemCpuLoad()`. That measures the
+recent CPU usage for the whole system. A value of 0.0 means that all CPUs were idle, while a value
+of 1.0 means that all CPUs were actively running 100% of the time.
```

Comment:
We should not confuse users with mentioning `1.0` here, when the rest of the docs are about percentage.

Reply:
Good catch, removed.